packages/components/tag/src/tag-list/tag-list.tsx

```diff
@@ -0,0 +1,31 @@
+import { type ReactNode, Children } from 'react';
+import { designTokens } from '@commercetools-uikit/design-system';
+import styled from '@emotion/styled';
+
+export type TTagListProps = {
+  children: ReactNode;
+};
+
+const TagListContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+`;
+
+const TagListItem = styled.div`
+  margin: ${designTokens.marginForTagList};
+`;
+
+const TagList = (props: TTagListProps) => {
+  return (
+    <TagListContainer>
+      {Children.map(props.children, (tag) => (
```

Comment:
Do we want to allow any child that is not a Tag component? If not, it will be nice to warn the user to use only the Tag component with `TagList`.

Reply:
<img width="429" alt="Screenshot 2023-04-17 at 15 21 21" src="https://user-images.githubusercontent.com/52276952/232516545-4689acb2-c8bb-4743-899a-e69eb7b8dab0.png">

Reply:
Valid Point @CarlosCortizasCT